core/src/main/java/io/keen/client/java/http/UrlConnectionHttpHandler.java

```diff
@@ -67,6 +68,11 @@ protected void sendRequest(HttpURLConnection connection, Request request) throws
         connection.setRequestProperty("Accept", "application/json");
         connection.setRequestProperty("Authorization", request.authorization);
 
+        // If a different HttpHandler is used, we won't get this header. We would need to refactor
```

Comment:
Thanks for the note here. I'm fine with this limitation, but it'll be good to know this if we ever get confused by missing headers in the future.